prismic/api.py

```diff
@@ -131,32 +131,36 @@ def query(self, q, ref=None, page_size=None, page=None, orderings=None, after=No
             form.after(after)
         if fetch_links is not None:
             form.fetch_links(fetch_links)
+        if lang is not None:
+            form.lang(lang)
         return form.query(q).submit()
 
-    def query_first(self, q, ref=None):
-        documents = self.query(q, ref, page_size=1, page=1).documents
+    def query_first(self, q, ref=None, lang=None):
+        documents = self.query(q, ref, page_size=1, page=1, lang=lang).documents
         if len(documents) > 0:
             return documents[0]
 
-    def get_by_uid(self, type, uid, ref=None):
-        return self.query_first(predicates.at('my.' + type + '.uid', uid), ref)
+    def get_by_uid(self, type, uid, ref=None, lang='*'):
```

Comment:
we forbid to use a wilcard for a getByUID since uids are defined per language.
https://github.com/prismicio/prismic-javascript/blob/master/src/ResolvedApi.ts#L199
you should implement the same logic there

Reply:
And add a test to make sure you can't query without a lang